server/controllers/userController.js

```diff
@@ -35,9 +35,37 @@ const createUser = async (req, res) => {
   return res.status(200).json(createdUser);
 }
 
-const updateUser = () => {
-  // TODO 2 - Actualizar información del usuario (Rocio)
-}
+const updateUser = (async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { first_name, last_name, email, password, age, photo, phone_number, gender, identity_document, status, birth_date } = req.body;
+    const selectedUser = await user.findByPk(id);
+
+    if (selectedUser === null) {
+      res.status(400)
```

Comment:
Aquí hace falta un return. De lo contrario el proceso va a pasar a la siguientes lineas. Cuando mandas la respuesta y aún hay líneas a ejecutar después se van a ejecutar. 

```suggestion
     return res.status(400)
```